src/identity.js

```diff
@@ -1273,6 +1273,17 @@ export default function Identity(mpInstance) {
              * @param {Function} [callback] A callback function that is invoked when the user audience request completes
              */
             getUserAudiences: function(callback) {
+                // user audience API is feature flagged
+                if (
+                    !mpInstance._Helpers.getFeatureFlag(
+                        Constants.FeatureFlags.AudienceAPI
+                    )
+                ) {
+                    mpInstance.Logger.error(
+                        Constants.Messages.ErrorMessages.AudienceAPINotEnabled
```

Comment:
Nit: In the top of the file, we destructure these nested constants, so this can be broken out like so:

```
// Top of file:
const ErrorMessages = Constants.Messages;

// Then here:
mpInstance.Logger.error(ErrorMessages.AudienceAPINotEnabled);
```